queries.sql

```diff
@@ -1,3 +1,28 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
\ No newline at end of file
+SELECT * FROM animals WHERE name LIKE '%mon' ;
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-01' ;
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3 ;
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu' ;
+SELECT name, escape attempts FROM animals WHERE weight_kg > 10.5 ;
+SELECT * FROM animals WHERE neutered = true ;
+SELECT * FROM animals WHERE name NOT IN ('Gabumon') ;
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 ;
+BEGIN ;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon' ;
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = '';
+COMMIT
+DELETE FROM animals WHERE date_of_birth > '2022-01-01' ;
+SAVEPOINT mult;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT mult;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
```

Comment:
- [ ] Kindly begin this transaction before you perform the `DELETE` query :pray:

Reply:
## OPTIONAL 
* I will suggest in the future you specify in your queries which part of your database you need data from so that your queries are efficient, for example rather than  select all animals for verification `SELECT * FROM animals;` you can target the specific part of the table needed in this case the species as this was what was updated so your query would look like this `SELECT species from animals;` now you only select animals based on their species this is more efficient